docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -351,8 +350,8 @@ front-end   10.110.250.153   <nodes>       80:30001/TCP   59s
 ```
 
 It takes several minutes to download and start all the containers, watch the
-output of `kubectl get pods -n sock-shop` to see when they're all up and
-running.
+output of `kubectl get pods -n sock-shop && kubectl get pods -n zipkin` to see 
+when they're all up and running.
```

Comment:
Where and how does zipkin get installed?  I don't see that in the previous yaml.  And I don't see where the namespace gets created...